templates/home/_fsf_python.html

```diff
@@ -34,6 +34,6 @@ <h4>Here’s how offlineimap uses it:</h4>
 
   <div class="p-flow-details__continue">
     In just a few steps, you’ll have an example Python app in the Snap Store.
-    <a class="p-button--positive is-inline" href="https://docs.snapcraft.io/build-snaps/python">Continue</a>
```

Comment:
There is also the same link on home page in languages section that could be updated for consistency.